src/controllers/company.number.controller.ts

```diff
@@ -7,7 +7,7 @@ import {COMPANY_NOT_FOUND, COMPANY_NUMBER_TOO_LONG,
     INVALID_COMPANY_NUMBER, NO_COMPANY_NUMBER_SUPPLIED} from "../model/error.messages";
 import {createGovUkErrorData, GovUkErrorData} from "../model/govuk.error.data";
 import {PROMISE_TO_FILE_CHECK_COMPANY} from "../model/page.urls";
-import {COMPANY_NUMBER} from "../model/template.paths";
+import {Templates} from "../model/template.paths";
```

Comment:
So are we going do format these import like { this } or {this}? It should be consistent throughout the service

Reply:
{ xxxx } seems to be the standard https://www.typescriptlang.org/docs/handbook/modules.htmlhttps://www.typescriptlang.org/docs/handbook/modules.html